### DISCUSSION TITLE
icub2.7 camera model number

### DISCUSSION THREAD

**Discussion by jimingda** [robotology/community#627] [Q&A] [answered]
Hi

Where can i get the exact product model number of the iCub2.7 camera ?
The info i get currently is that they are basler cameras from the following part.
https://icub-tech-iit.github.io/documentation/upgrade_kits/head_4k/support/?h=camera#upgrade-kit

**Comment by maggia80** [comment, marked as answer]
Hi @jimingda the cameras mounted on your robot are the: pointgrey dragonfly2 640x480 30fps:
[POINTGREY - Dragonfly2.pdf](https://github.com/robotology/community/files/11543544/POINTGREY.-.Dragonfly2.pdf)
